examples/cap-extSL.jl

```diff
@@ -0,0 +1,26 @@
+# ~/~ begin <<docs/src/ca-with-production.md#examples/cap-extSL.jl>>[init]
+using CarboKitten.CaProd
+using DataFrames
+using CSV
+using Interpolations
+#datasl = CSV.read("data/bs92-sealevel-curve.csv", DataFrame, header=1)
+
+DEFAULT_INPUT = CaProd.Input(
+    sea_level = t -> CSV.read("data/all-sealevel/sl.csv", DataFrame, header=1)[!,1][round(Int,t*1000)+1],
```

Comment:
This amounts to a nearest neighbour interpolation. It's probably better to use a linear interpolator from the `Interpolations` package.